salt/modules/zpool.py

```diff
@@ -907,12 +925,7 @@ def split(zpool, newzpool, **kwargs):
     properties : dict
         additional pool properties for newzpool
 
-    CLI Example:
-
-    .. code-block:: bash
-
-        salt '*' zpool.split datamirror databackup
-        salt '*' zpool.split datamirror databackup altroot=/backup
+    .. versionadded:: Oxygen
```

Comment:
I know this wasn't your code, but can you change this to 2018.3.0? We _just_ updated these references in the oxygen branch. :)

Reply:
Done, updates the references in all files I touched.